app/Services/Manager.php

```diff
@@ -92,6 +92,77 @@ public function exportCSV($model, $hasReportback = false)
     }
 
     /**
+     * Build leaderboard data for a given Competition
+     *
+     * @param Competition $competition
+     * @param User $users
+     * @return array $leaderboard
+     */
+    public function getLeaderboard($competition, $users)
+    {
+        $rows = [];
+
+        // Get all users in bulk
+        $parameters['ids'] = implode(',', $users->pluck('id')->all());
+        $users = $this->northstar->getAllUsers($parameters);
+        $users = collect($users)->keyBy('id')->all();
+
+        foreach ($users as $user) {
+            // For each user get the reportback details
+            $reportback = $this->getUserActivity($user->id, $competition);
```

Comment:
ahh I see your note now 
> Also this doesn't get signups/reportbacks as a group. Since that was proving to be difficult I'll tackle that next, but at least we are getting leaderboards live.